_guide/onboarding-staff.md

```diff
@@ -44,73 +44,86 @@ Curious how some other people engage with the web?
 
 Our team has been working to extend the [CivicActions Accessibility](https://accessibility.civicactions.com/) site and integrate key elements into the [CivicActions Handbook](https://handbook.civicactions.com/). CivicActions believes in working in the open, so we also are leveraging tools on GitHub like the [Projects board](https://github.com/CivicActions/accessibility/projects/1) and [Discussions pages](https://github.com/CivicActions/accessibility/discussions).
 
-
 ### Trello card - Install a web accessibility browser extension
 
-Install an accessibility browser extension like [WebAim's WAVE Toolbar extension/plugin](https://wave.webaim.org/extension/) or [Microsoft's Accessibility Insights](https://accessibilityinsights.io/). Take the time to learn the basics of whichever one you choose because everyone has a role in producing accessible websites.
-  * [WAVE](https://www.youtube.com/watch?v=ITUDiTgAZY0&t=270s)
-  * [Accessibility Insights](https://www.youtube.com/watch?v=U6NY8Cxym5g)
+Install an accessibility browser extension. Take the time to learn the basics of whichever one you choose because everyone has a role in producing accessible websites.
+
+* [Deque's axe DevTool for Chrome](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd) - [Youtube: Intro to axe DevTools](https://www.youtube.com/watch?v=iRGB40c_YJc)
+* [WebAim's WAVE](https://wave.webaim.org/) - [Youtube: Intro to WAVE](https://www.youtube.com/watch?v=ITUDiTgAZY0&t=270s)
+* [Microsoft's Accessibility Insights](https://accessibilityinsights.io/) - [Youtube: Intro to Accessibility Insights](https://www.youtube.com/watch?v=U6NY8Cxym5g)
+
+### Trello card - Install a color contrast checker
+
+Install a color contrast checker browser extension, such as [ColorZilla for Chrome](https://chrome.google.com/webstore/detail/colorzilla/bhlhnicpbhignbdhedgjhgdocnmhomnp), [ColorZilla for Firefox](https://addons.mozilla.org/en-US/firefox/addon/colorzilla/) or use a free site like [Coolors Color Contrast Checker](https://coolors.co/contrast-checker/112a46-acc8e5)
+
+### Trello card - Developers, install an accessibility linter in your text editor or Integrated Development Environment (IDE)
+
+Install an accessibility linter in your text editor, such as [axe Accessibility Linter for VS Code - recommended](https://marketplace.visualstudio.com/items?itemName=deque-systems.vscode-axe-linter). Please note, an accessiblity linter is required for front end dev work.
```

Comment:
Do we have one for phpstorm or a link for other IDE's developers may use

Reply:
There is nothing like this that I have been able to find for PHP Storm